src/backend/parser/parse_partition_gp.c

```diff
@@ -1469,7 +1469,7 @@ generatePartitions(Oid parentrelid, GpPartitionDefinition *gpPartSpec,
 	 */
 	foreach(lc, gpPartSpec->partDefElems)
 	{
-		Node	   *n = lfirst(lc);
```

Comment:
It seems that variable n is used in the next IsA(n, ColumnReferenceStorageDirective). Do we have need to add this PG_USED_FOR_ASSERTS_ONLY?

Reply:
Looks like parts of this commit https://github.com/greenplum-db/gpdb-archive/commit/80576302ae02039c14c61d798b0631cb276a5790

oks, removed it